src/app/explore/explore.component.html

```diff
@@ -15,7 +15,7 @@
       [beneficiaryOptions]="beneficiaryOptions"
       [locationOptions]="locationOptions"
       [fundingOptions]="fundingOptions"
-      [selectedSortByOption]="searchService.getSelectedSortLabel()"
+      [selectedSortByOption]="searchService.selectedSortLabel"
```

Comment:
The prop type is `'Most raised' | 'Match funds remaining' | 'Relevance' | null` but you're passing a `string` which shouldn't be assignable - looks like we don't have SA working between the template and the prop type, but would be good if we could get them matching up.

Reply:
I've made a PR to your PR: https://github.com/thebiggive/donate-frontend/pull/1089

Reply:
@bdsl Sure. Not had time to test it in context but I've made a `components` change to make the prop non-nullable: https://github.com/thebiggive/components/pull/164

The types of the lib are checked independently and it should be safe to assign the narrower type to the wider one afaik. So I think these could probably be merged in either order / without necessarily waiting for a new `components` version.

Reply:
In theory narrow types can be assigned to wide, not vise versa. So your PR 164 should be a BC break - but I don't think it matters too much while these aren't being checked and yes they can be merged in either order. Really 1089 should be changed to remove the 'null' part from it now.